client/cli/src/commands/run_cmd.rs

```diff
@@ -267,9 +267,9 @@ pub struct RunCmd {
 
 	/// Specify remote authority address
 	///
-	/// Ex: `/ip4/127.0.0.1/tcp/<port>`
+	/// Ex: `http://localhost`
```

Comment:
Please change comment, port is misisng.

Reply:
There is default port (`80`) and its not required to explicitly specify it. I'll change example to show how to use non default port ;)